front/lib/api/assistant/relation_override.ts

```diff
@@ -8,14 +8,17 @@ import {
   AgentUserRelation,
 } from "@app/lib/models/assistant/agent";
 
+/**
+ *
+ * @returns Map of agentId to relation override
+ */
 export async function getAgentRelationOverridesForUser(
```

Comment:
Is the query behind that function indexed? It now lies on the critical path of loading agents